### PR TITLE
Fix ProgressIndicators

### PR DESCRIPTION
**Pull request by HansMuller**
Set the the ProgressIndicator's CustomPaint token to the ProgressIndicator's value.

### CODE REVIEW

**Comment by abarth** [comment]
LGTM but we should write a test for this....

**Comment by HansMuller** [comment]
Good point. No more regressions!  I will try and write a test...

File: sky/unit/test/widget/build_utils.dart [hunk 1]
```diff
@@ -64,6 +65,20 @@ class WidgetTester {
...
+      ContainerLayer root = layer;
+      Layer child = root.firstChild;
+      while(child != null) {
+        result.addAll(_layers(child));
+        child = child.nextSibling;
+      }
+    }
+    return result;
+  }
+  List<Layer> layers() => _layers(_renderView.layer);
```
Comment (abarth) [inline sky/unit/test/widget/build_utils.dart]: List<Layer> layers()    ->    List<Layer> get layers

File: sky/unit/test/widget/build_utils.dart [hunk 2]
```diff
@@ -114,4 +129,13 @@ class WidgetTester {
     Component.flushBuild();
     RenderObject.flushLayout();
   }
+
+  void pumpPaintFrame(WidgetBuilder builder) {
```
Comment (abarth) [inline sky/unit/test/widget/build_utils.dart]: Can we just make pumpFrame do this?  I'd like to avoid having a separate function.

**Comment by abarth** [comment]
LGTM

File: sky/unit/test/widget/build_utils.dart (hunk 2 above)
Comment (HansMuller) [inline sky/unit/test/widget/build_utils.dart]: Perhaps that's possible. At the moment, doing so causes global_key_test.dart to fail:
```
rendering/object.dart': Failed assertion: line 710 pos 12: '!_needsCompositingBitsUpdate' is not true.

I'll add a TODO for now.

File: sky/unit/test/widget/build_utils.dart (hunk 1 above)
Comment (HansMuller) [inline sky/unit/test/widget/build_utils.dart]: OK

File: sky/unit/test/widget/build_utils.dart (hunk 2 above)
Comment (abarth) [inline sky/unit/test/widget/build_utils.dart]: Ok.  I can work on fixing that.

**Pull request merged**